udata/core/dataset/models.py

```diff
@@ -160,6 +155,14 @@ def is_available(self):
         return self.check_availability(group=None)
 
 
+class Resource(ResourceMixin, WithMetrics, db.EmbeddedDocument):
+    id = db.AutoUUIDField()
+    owner = db.ReferenceField('User')
```

Comment:
Does the resource still have an owner ? I think this one was only for community resources.